deliverables/android.html

```diff
@@ -1,23 +1,125 @@
+
 <!DOCTYPE html>
 <html>
 <head>
   <title>Ignition Android</title>
+  <!--<link rel="stylesheet" type = "text/css" href = "android.css"/>-->
   <style type='text/css'>
     div div {
       position: relative;
     }
+div {margin: o; padding: 0;}
+div div {background: #0F21FB; position: relative;}
```

Comment:
Since `background` and `position` are very different properties, I'd recommend splitting them to separate lines.  Later on where you put height and width on the same line, that makes more sense to me.  It's all personal taste though.  

``` css
div div {
  background: #0F21FB;
  position: relative;
}
```